src/components/Dashboard/Rewards.tsx

```diff
@@ -96,7 +96,8 @@ export default function Rewards() {
               text={tokenSymbol ? tokenSymbol + ' ' + t`tokens` : t`Tokens`}
               tip={
                 `${tokenSymbol ? tokenSymbol + ' ' + t`ERC20` : t`Tokens`}` +
```

Comment:
**suggestion**: let's wrap this whole thing in a `t`

Reply:
Remember the problem we were having with `t`s within `t`s ? Not a great solution but since I couldn't figure a way around this I left it like that

<img width="624" alt="Screen Shot 2022-02-05 at 8 22 37 pm" src="https://user-images.githubusercontent.com/96150256/152638197-6a792bc6-3c62-40c3-afe4-75ff2304ff8e.png">